RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -1211,6 +1211,15 @@ namespace mkfit {
                   itrack, layer_of_hits.is_barrel(), charge_pcm[itrack], Hit::minChargePerCM(), propPar, m_msErr);
             }
 
+            // Select only SiStrip hits with cluster size <8
+            if (!layer_of_hits.is_pixel()) {
+              if (layer_of_hits.refHit(m_XHitArr.At(itrack, hit_cnt, 0)).spanRows() >= 8)
+                isCompatible = false;
+            }
+            //else { // Uncomment to apply analogous cut on pixel cluster size
+            //  if (layer_of_hits.refHit( m_XHitArr.At(itrack, hit_cnt, 0) ).spanRows() >= 8 || layer_of_hits.refHit( m_XHitArr.At(itrack, hit_cnt, 0) ).spanCols() >= 8) isCompatible = false;
+            //}
```

Comment:
can this be made configurable?

Reply:
I'm not sure it makes much sense to configure a cut if we wish to keep 3 bits.
Aside of how much it makes sense, it should be possible, although it may be more difficult to make it configurable, similarly to minPtCut.

Reply:
I have updated the PR as discussed privately (also with @cerati).
Now, the cut is configurable, and the number of hits has been extended (from 3 to 5) to allow for looser cut values.
Note that the cut is currently configurable per iteration (also being the only straight-forward option for configurability within CMSSW, as of today).